tests/integration/web/info/room_test.py

```diff
@@ -8,6 +8,11 @@
 from nav.web.info.room.views import ADD_SENSOR_MODAL_ID
 
 
+class TestRoomInfoView:
+    def test_room_info_url_should_allow_slashes(self, client):
+        assert reverse('room-info', args=['TEST/SLASH'])
```

Comment:
Same comment as before, it would be good to actually have a room with slash in the name and then check for a valid response